Gemfile.lock

```diff
@@ -46,22 +46,22 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (1.2.0)
       rails (~> 6.1)
-    activejob (6.1.1)
-      activesupport (= 6.1.1)
+    activejob (6.1.3.1)
+      activesupport (= 6.1.3.1)
       globalid (>= 0.3.6)
-    activemodel (6.1.1)
-      activesupport (= 6.1.1)
-    activerecord (6.1.1)
-      activemodel (= 6.1.1)
-      activesupport (= 6.1.1)
-    activestorage (6.1.1)
-      actionpack (= 6.1.1)
-      activejob (= 6.1.1)
-      activerecord (= 6.1.1)
-      activesupport (= 6.1.1)
-      marcel (~> 0.3.1)
-      mimemagic (~> 0.3.2)
-    activesupport (6.1.1)
+    activemodel (6.1.3.1)
+      activesupport (= 6.1.3.1)
+    activerecord (6.1.3.1)
```

Comment:
### [activerecord](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.1.1 → 6.1.3.1

[_[change-log](https://github.com/rails/rails/blob/v6.1.3.1/activerecord/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.1.3.1/activerecord)_]

:tada: Patched vulnerabilities:

- [CVE-2021-22880](https://nvd.nist.gov/vuln/detail/CVE-2021-22880)
  **Possible DoS Vulnerability in Active Record PostgreSQL adapter**

  
  URL: https://groups.google.com/g/rubyonrails-security/c/ZzUqCh9vyhI

  There is a possible DoS vulnerability in the PostgreSQL adapter in Active Record. This vulnerability has been assigned the CVE identifier CVE-2021-22880.  Versions Affected:  >= 4.2.0 Not affected:       < 4.2.0 Fixed Versions:     6.1.2.1, 6.0.3.5, 5.2.4.5  Impact ------ Carefully crafted input can cause the input validation in the "money" type of the PostgreSQL adapter in Active Record to spend too much time in a regular expression, resulting in the potential for a DoS attack.  This only impacts Rails applications that are using PostgreSQL along with money type columns that take user input.  Workarounds ----------- In the case a patch can't be applied, the following monkey patch can be used in an initializer:  ``` module ActiveRecord   module ConnectionAdapters     module PostgreSQL       module OID # :nodoc:         class Money < Type::Decimal # :nodoc:           def cast_value(value)             return value unless ::String === value              value = value.sub(/^\((.+)\)$/, '-\1') # (4)             case value             when /^-?\D*+[\d,]+\.\d{2}$/  # (1)               value.gsub!(/[^-\d.]/, "")             when /^-?\D*+[\d.]+,\d{2}$/  # (2)               value.gsub!(/[^-\d,]/, "").sub!(/,/, ".")             end              super(value)           end         end       end     end   end end ```






<details>
<summary>Commits</summary>

A change of **80** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.1.1...v6.1.3.1).

These are the first 10 commits:
- (689d096) [Merge pull request #41049 from rails/revert-39321-fix_update…](https://github.com/rails/rails/commit/689d0963e0e79f5daf0d3962c8da410b501ebaec)
- (5284a16) [Merge pull request #41030 from eileencodes/fix-preventing-wr…](https://github.com/rails/rails/commit/5284a165b8e8821fcfca62dac360bd9ed7a674b4)
- (67fd5cb) [Fix failing tests introduced by #41030](https://github.com/rails/rails/commit/67fd5cb3efe6cbd31ad88261abe9a03f186811c5)
- (9f68683) [Merge pull request #41050 from alea12/update-actioncable-doc…](https://github.com/rails/rails/commit/9f68683e57adbe7aad544e3c1c2c9e264517ca66)
- (044236e) [Merge pull request #41052 from bogdanvlviv/preload_links_hea…](https://github.com/rails/rails/commit/044236e3beec448d78bdcc3dd5209c6e90a8fc50)
- (90339be) [Merge pull request #41063 from kamipo/fix_where_with_polymor…](https://github.com/rails/rails/commit/90339be70c770e2f975e099c80f0b4462ad63297)
- (9cc99d7) [Merge pull request #40704 from alpaca-tc/sandbox_multiple_da…](https://github.com/rails/rails/commit/9cc99d70c991b586ec9e74fbc391b599eba37bb8)
- (12bb9d3) [Merge pull request #41082 from kamipo/fix_update_optimistic_…](https://github.com/rails/rails/commit/12bb9d32f56883914abcd98fd72e3c68c444808d)
- (acb6631) [Merge pull request #41090 from kamipo/fix_find_by_with_custo…](https://github.com/rails/rails/commit/acb6631cd99cdfe7db356773ef74cad7cbb570ed)
- (3f4fde4) [Merge pull request #41098 from kamipo/fix_source_type_with_s…](https://github.com/rails/rails/commit/3f4fde4d9f804140be8304b524792c052e3d1024)


</details>